web_editor/index.html

```diff
@@ -0,0 +1,66 @@
+<html>
+  <head>
+    <script src="https://cdn.jsdelivr.net/npm/sortablejs@latest/Sortable.min.js"></script>
+
+    <style>
+      .cards-row {
+        display: flex;
+        flex-direction: row;
+      }
+
+      .card {
+        outline: solid;
+        padding: 15px;
+      }
+
+      .card-being-dragged {
+        background-color: #c8ebfb;
+      }
+    </style>
+
+    <script type="module" src="js/main.js"></script>
+  </head>
+  <body>
+    <fieldset>
+      <legend>OSM way ID</legend>
+      <input type="text" id="osm_way_id" value="804788513" />
+    </fieldset>
+    <input type="button" value="Edit" onclick="window.start_editing();" />
+
+    <div id="cards" class="cards-row">
+      <div class="card">Item 1</div>
+    </div>
+
+    <div style="border: 3px solid; padding 20px; margin-top: 30px">
+      <div
+        id="delete"
+        class="cards-row"
+        style="
+          float: left;
+          margin-right: 50px;
+          border: 2px dashed red;
+          min-height: 100px;
+          padding: 30px;
+        "
+      >
+        Delete lane
+      </div>
+
+      <div id="toolbox" class="cards-row" style="float: left">
+        <!-- TODO Can I just make up / overload the value attribute here? -->
```

Comment:
I'm just sort of stuffing an "enum" in here as `value`, then later in the JS turning this into JSON data for a lane. Is that a half-reasonable pattern to start with?